### PR TITLE
Update geoprocessing script for latest component schema

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13424

This script continues to be a helper for giving Andrew data he needs for the Mobility Annual Plan. I would definitely want to refactor this before using it in an ETL. Ultimately I think we'll want to make a DB view that does most of the munging.

## Testing
**URL to test:**  Local

1. Start the cluster + editor
2. navigate to this this script: `/atd-moped/moped-toolbox/component_geo_processing`
3. Save `secrets_template.py` as `secrets.py`
4. Using a Python environment with `requests`  you can run the script:

```shell
 python create_components_geojson.py -e local
```
5. Inspect the contents of the `/_output` directory. you can copy/paste the geojson into http://geojson.io/ if you like.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by johnclary** [comment]
👋 i'm really not asking for a code review unless you want to 🕵️ i've already handed off the data to Andrew.

**Comment by mddilley** [comment]
@johnclary just self-requested on this one because of to the accidental assignees

**Comment by johnclary** [comment]
oops thanks @mddilley!

File: moped-toolbox/component_geo_processing/secrets.py [hunk 1]
```diff
@@ -3,6 +3,6 @@
         "local": "http://localhost:8080/v1/graphql",
     },
     "hasura_graphql_admin_secret": {
-        "local": "DuMmyApiKeyHFVOVto19otC1wX6sP2N0VSKrCD70L10B7Sm525ZR6L672i2F79M9!",
+        "local": "hasurapassword",
```
Comment (chiaberry) [inline moped-toolbox/component_geo_processing/secrets.py:6]: 👍🏼

File: moped-toolbox/component_geo_processing/create_components_geojson.py [hunk 2]
```diff
@@ -161,18 +179,22 @@ def main(env):
...
+                f
+                for f in data["project_geography"]
+                if f["component_id"] == proj_component["project_component_id"]
+            ]
             geometry = merge_geoms(features)
             properties = get_component_properties(proj_component["moped_components"])
+            add_council_districts(properties, features)
             add_project_properties(properties, project_data, PROJECT_LIST_KEYS)
             add_project_tags(properties, proj["moped_proj_tags"])
+            add_subcomponents(properties, proj_component)
```
Comment (mddilley) [inline moped-toolbox/component_geo_processing/create_components_geojson.py:197]: ah, cool, this is a helper now 👍

**Review by mddilley — APPROVED** [review]
Code changes make sense to me and approving because I think that the data would have bounced back by now if it didn't work out for Andrew. Thanks! 🚀

**Review by chiaberry — APPROVED** [review]
agree with Mike, I bet if there was an issue with the output, you would have heard by now

**Pull request merged**